Prevent definition of same gvk in custom resource configuration (#1810)
kubernetes/kube-state-metrics: docs/customresourcestate-metrics.md

Two flags can be used:

* `--custom-resource-state-config "inline yaml (see example)"` or
* `--custom-resource-state-config-file /path/to/config.yaml`

Both flags can be provided multiple times to load multiple configurations.

> When multiple entries for the same resource exist, kube-state-metrics will refuse to start.

Reviewer: ```suggestion
When multiple entries for the same resource exist, kube-state-metrics will exit with an error.
```
